### PR TITLE
added objectives, survey links, and thanks

### PR DESCRIPTION
**Pull request by raynamharris**
I tried to beef this section up since its so important. 
I added 
- some objectives and a questions,
- the new survey monkey link
- a little paragraph about thanks. 

I like to end by thanking everyone so they leave feeling good about themselves :)

### CODE REVIEW

File: _episodes/21-summarize.md [hunk 1]
```diff
@@ -5,7 +5,15 @@ teaching: 30
 exercises: 15
 questions:
 - "What have we learned?"
+- "What do we do next?"
+objectives:
+- Understand final steps required to qualify as an instructor
+- Evaluate the utility of the insturctor training workshop
```
Comment (lexnederbragt) [inline _episodes/21-summarize.md]: instructor

File: _episodes/21-summarize.md [hunk 2]
```diff
@@ -18,8 +26,25 @@ The final three steps in qualifying as an instructor are to:
...
+We frequently also ask for summary feedback at the end of each day. The instructors ask the learners to alternately give one positive and one negative point about the day, without repeating anything that has already been said. This requirement forces people to say things they otherwise might not: once all the “safe” feedback has been given, participants will start saying what they really think. Minute cards are anonymous; the alternating up-and-down feedback is not. Each mode has its strengths and weaknesses, and by providing both, we hope to get the best of both worlds.
+
 > ## One Up, One Down
 >
 > Provide [one up, one down]({{ site.root }}/20-practices/) feedback
 > on the entire two-day course.
 {: .challenge}
+
+**Instructor training post-workshop survey.**
+Just like in our regular workshops, we are collecting post-instructor-training-workshop feedback. Your participation will help us evaluate the efficacy of our workshops and improve the conent and delivery of the lesson materials.
```
Comment (lexnederbragt) [inline _episodes/21-summarize.md]: content

File: _episodes/21-summarize.md [hunk 3]
```diff
@@ -18,8 +26,25 @@ The final three steps in qualifying as an instructor are to:
...
+
+> ## Survey
+>
+> Complete this [~5 min post-workshop survey](https://www.surveymonkey.com/r/post-instructor-training).
+{: .challenge}
+
+## Thanks!
+
+The instructors would like to thank you for your enthusiatic participation in this insturctor-training workshop. We hope it was a valuable and enjoyable experierence. We look forward to your continued involvment in the Software and Data Carpentry communities!
```
Comment (lexnederbragt) [inline _episodes/21-summarize.md]: enthusiastic, experience

File: _episodes/21-summarize.md (hunk 1 above)
Comment (jttkim) [inline _episodes/21-summarize.md]: ??

**Pull request merged**